app/src/renderer/components/governance/PageProposal.vue

```diff
@@ -4,24 +4,27 @@
       <tm-balance />
       <tool-bar />
     </template>
+
     <tm-data-error v-if="!proposal" />
+
     <template v-else>
       <div class="page-profile__header page-profile__section proposal">
-        <div class="column page-profile__header__info">
-          <div class="row page-profile__header__name">
-            <div class="top column">
-              <div class="page-profile__status-and-title">
-                <span
-                  v-tooltip.top="status.message"
-                  :class="status.color"
-                  class="page-profile__status"
-                />
-                <div class="page-profile__header__name__title">
-                  {{ proposal.title }} {{ `(#` + proposalId + `)` }}
-                </div>
-              </div>
+        <div class="row">
+          <h2 class="proposal-id">#{{ proposalId }}</h2>
+          <div class="page-profile__header__info">
+            <div class="page-profile__status-and-title">
+              <span
+                v-tooltip.top="status.message"
+                :class="status.color"
+                class="page-profile__status"
+              />
+              <h2 class="page-profile__title">{{ proposal.title }}</h2>
+              <h3 v-if="config.devMode">
+                Proposer: {{ proposal.proposer || "Fede" }}
```

Comment:
lol what?

Reply:
lol, missed that XD

Reply:
💍 👰

Reply:
i do believe that this will be available in the next release of the SDK, right?